package.json

```diff
@@ -13,7 +13,8 @@
     "build": "npx tsc",
     "clean": "rm -rf build/",
     "prepare": "npm run clean && npm run build",
-    "lint": "npx eslint src"
+    "lint": "npx eslint src",
+    "postinstall": "npm r build"
```

Comment:
npm remove build ? what's this about?

Reply:
should've been `run`, but will remove this entirely